library/core/src/main/java/com/google/android/exoplayer2/audio/Ac4Util.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.exoplayer2.audio;
+
+import com.google.android.exoplayer2.C;
+import com.google.android.exoplayer2.Format;
+import com.google.android.exoplayer2.drm.DrmInitData;
+import com.google.android.exoplayer2.util.MimeTypes;
+import com.google.android.exoplayer2.util.ParsableBitArray;
+import com.google.android.exoplayer2.util.ParsableByteArray;
+import java.nio.ByteBuffer;
+
+/**
+ * Utility methods for parsing AC-4 frames, which are access units in AC-4 bitstreams.
+ */
+public final class Ac4Util {
+
+  /**
+   * Holds sample format information as presented by a syncframe header.
+   */
+  public static final class SyncFrameInfo {
+
+    /**
+     * The sample mime type of the bitstream is {@link MimeTypes#AUDIO_AC4}.
+     */
+    public final String mimeType;
```

Comment:
Looks like this is fixed and only referred to in Ac4Reader. To make it clear that only one MIME type is possible, could you inline the constant where this field is referred to, then remove this field?

Reply:
OK

Reply:
I think `Ac3Util` is different to this one as it handles both `MimeTypes.AUDIO_E_AC3` and `MimeTypes.AUDIO_AC3` and needs to signal which one is used, which this case seems to have only one possible MIME type.

Reply:
You are right. I'll update the code.